sass/bhclub.scss

```diff
@@ -113,12 +124,73 @@ nav {
 
 // Main Content
 main {
-    padding: 3.5rem;
+    padding: 1.75rem 3.5rem;
     padding-left: calc(25% + 3.5rem); // pushes it outside of the nav while allowing for backgrounds to stretch underneath
 }
 
 section {
-    padding: 5rem 0;
+    padding: 1.75rem 0;
+    min-height: 100vh;
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: center;
+}
+
+// About Section
+#about p {
+    width: 60%;
+}
```

Comment:
As written, this cascades into the cards' body text and adds an awkward blank space on the right end of each:

![blankspace](https://user-images.githubusercontent.com/16471311/127257565-620dfff4-21dd-42c7-9e6a-a926a054f2f1.png)

This isn't compliant with the Figma mockup:

![image](https://user-images.githubusercontent.com/16471311/127257656-ff58fb4c-3259-4eaf-a598-1ffe4a1aa085.png)

Here's a possible fix:

```suggestion
#about > p {
    width: 60%;
}
```